charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -107,6 +107,10 @@ k8sTagClusterId: ""
 # region: us-east-1
 region: ""
 
+# Additonal environment variables for the controller
+controller:
```

Comment:
I guess this is fine as @krmichel will do this regardless.

Can you update the chart version though (just the patch field)? Otherwise releaser fails.

Reply:
Thanks for checking. I've done this now

Reply:
Mine will be slightly different as I have added the ability to set env vars for each of the containers in the controller pod.
https://github.com/krmichel/aws-ebs-csi-driver/commit/e0261ec85e58e06a00b00e76eda19175798eea6f#diff-cd438cefb202fb6fb653f659771ea635aa04dd99bb6d0902a0ab8eabebcf0a1eR68